services/static-webserver/client/source/class/osparc/data/model/Node.js

```diff
@@ -1340,7 +1340,13 @@ qx.Class.define("osparc.data.model.Node", {
 
       // ping for some time until it is really reachable
       try {
+        if (osparc.utils.Utils.isDevelopmentPlatform()) {
+          console.log("Connecting: about to fetch ", srvUrl);
+        }
         const response = await fetch(srvUrl);
+        if (osparc.utils.Utils.isDevelopmentPlatform()) {
+          console.log("Connecting: fetch's response ", response);
```

Comment:
TYPO: `fetches response`

Reply:
Isn't the response of the fetch, fetch's response?